authors/apps/authentication/validation.py

```diff
@@ -0,0 +1,20 @@
+"""
+Password validation will be handled in this file.
+
+"""
+import re
+
+
+class Validators:
```

Comment:
Nice work @NicholusMuwonge.

Please consider a different approach rather than having a class with one static method. Kindly look at serializers.py for a start.

Reply:
> Kindly include a check for whether an email exists.

hey @RonKbS this is already handled in the exceptions since the email is a unique field, so when you try to signup twice, you'll get an appropriate response, hence there's a test for it too.

Reply:
Let me also change the validation to the serializers.py as recommended

Reply:
I mean exists generally and not whether it has already been used to register with our application.